src/common/pico_time/time.c

```diff
@@ -253,8 +253,9 @@ static void alarm_pool_dump_key(pheap_node_id_t id, void *user_data) {
 #endif
 }
 
-static int64_t repeating_timer_callback(alarm_id_t id, void *user_data) {
+static int64_t repeating_timer_callback(__unused alarm_id_t id, __unused void *user_data) {
```

Comment:
Why the `__unused` on `user_data`? It seems to be used on the next line?
`repeating_timer_t *rt = (repeating_timer_t *)user_data;`

Reply:
a little trigger happy i guess